src/backend/tests/test_friendly_rule.py

```diff
@@ -20,10 +44,40 @@ def test_arrest_is_unaffected_if_conviction_is_older():
     expunger.run()
 
     assert arrest.expungement_result.time_eligibility.status is EligibilityStatus.ELIGIBLE
-    assert arrest.expungement_result.time_eligibility.date_will_be_eligible == None
+    assert arrest.expungement_result.time_eligibility.date_will_be_eligible == arrest.disposition.date
```

Comment:
I suggest passing a separate charge date for this charge, so that 
`     assert arrest.expungement_result.time_eligibility.date_will_be_eligible == arrest.disposition.date`
fails as it should, but 
`     assert arrest.expungement_result.time_eligibility.date_will_be_eligible == arrest.date`
passes

Reply:
Unfortunately this requires changing `ChargeFactory.create` and as you know that has cascading effects.